README.md

```diff
@@ -26,62 +26,83 @@ The purpose of this repository is to contain libraries which users can import an
+
+#### Math
+
 - [Fixed Point Number](./libs/fixed_point/) is an interface to implement fixed-point numbers.
+- [Signed Integers](./libs/signed_integers/) is an interface to implement signed integers.
+
+#### Data Structures
+
 - [Queue](./libs/queue/) is a linear data structure that provides First-In-First-Out (FIFO) operations. 
-- [Token](./libs/token/) provides helper functions for the [SRC-20](https://github.com/FuelLabs/sway-standards/tree/master/standards/src_20), [SRC-3](https://github.com/FuelLabs/sway-standards/tree/master/standards/src_3), and [SRC-7](https://github.com/FuelLabs/sway-standards/tree/master/standards/src_7) standards.
-- [Pausable](./libs/pausable/) allows contracts to implement an emergency stop mechanism.
 
 ## Using a library
 
-To import the Merkle Proof library the following should be added to the project's `Forc.toml` file under `[dependencies]` with the most recent release:
+To import a library, a dependency should be added to the project's `Forc.toml` file under `[dependencies]`. The following shows adding the Merkle Proof Library.
 
 ```rust
 merkle_proof = { git = "https://github.com/FuelLabs/sway-libs", tag = "v0.1.0" }
 ```
 
+> **NOTE:** 
+> Be sure to set the tag to the latest release.
+
 You may then import your desired library in your Sway Smart Contract as so:
 
-```rust
+```sway
 use merkle_proof::<library_function>;
 ```
 
-For example, to import the Merkle Proof library use the following statement:
+For example, to import the `verify_proof()` function use the following statement:
 
-```rust
-merkle_proof::binary_merkle_proof::verify_proof;
+```sway
+use merkle_proof::binary_merkle_proof::verify_proof;
 ```
 
 ## Running Tests
 
-In order to run the tests make sure you are in the tests folder of this repository `sway-libs/tests/<you are here>`.
+There are two sets of tests that should be run: inline tests and sdk-harness tests.
 
-Build the test projects:
+In order to run the inline tests, make sure you are in the `libs/` folder of this repository `sway-libs/libs/<you are here>`.
+
+Run the tests:
 
 ```rust
-forc build
+forc test
 ```
 
-> **Note**
-> This may take a while depending on your hardware, future improvements to Sway will decrease build times. After this has been run once, indiviual test projects may be built on their own to save time.
+Once these tests have passed, make sure you are in the `tests/` folder of this repository `sway-libs/tests/<you are here>`.
 
 Run the tests:
 
 ```
-cargo test
+forc test && cargo test
 ```
 
+> **NOTE:**
+> This may take a while depending on your hardware, future improvements to Sway will decrease build times. After this has been run once, indiviual test projects may be built on their own to save time.
+
 Any instructions related to using a specific library should be found within the README.md of that library.
 
-> **Note**
+> **NOTE:**
 > All projects currently use `forc v0.47.0`, `fuels-rs v0.46.0` and `fuel-core 0.20.8`.
```

Comment:
oh, just wondering about the sdk version here. do the latest releases work with that version? I thought that was out of date and incompatible

Reply:
I haven't gotten around to updating the SDK. The focus has been on the libraries themselves such that devs can use them for the hackathon. Another reason why I want to do as much inline testing as possible.